_posts/2021-12-10-start-a-workspace-by-entering-a-url.adoc

```diff
@@ -0,0 +1,123 @@
+---
+title: Start a workspace by entering a URL
+layout: post
+author: Florent Benoit
+description: >-
+  How to start a workspace by customizing the URL.
+categories: []
+keywords: ['GitHub', 'GitLab', 'Bitbucket', 'start', 'workspace']
+slug: /@florent.benoit/start-a-workspace-by-entering-a-url
+---
+
+== Starting a workspace by entering a URL
+
+As a user, you can create a workspace containing a clone of a Git repository by entering:
+
+* a custom URL in the address bar of your browser.
+* the URL of the Git repository on the {prod} dashboard page.
+
+=== Short URL syntax for cloning hosted Git repositories
+
+In the earlier versions of {prod}, the {prod-short} URL syntax began with `/f/?url=`, for example `pass:c,a,q[{prod-url}/f/?url=https://github.com/my/repository]`, but it was complex and error-prone. While that syntax is still supported, short syntax should be used as follows:
```

Comment:
This line is a good example of something that is perfectly fine in a blog post, but not in the docs.